### ISSUE TITLE
Using a High Resolution and a Windows internal scalation

### ISSUE THREAD

**Issue by HappyHippoSnack** [OpenBoard-org/OpenBoard#538]
When using a resolution around 3840 x 2160 and the internal scal of 100% all icons are really small. (Of course)

When the scalation is set higher (200% - 400%) all lines are blurry and not "sharp". 

Would be great, if there is a way of setting the scale in the options or try to get the ClearType thing of MS into the program.

**Comment by entity0x** [comment]
Yes, first thing after opening OpenBoard is that it requires UI scaling on 3840X2160(4K) displays. Would like to see this added, as right now it is unusable.
It may even affect the right side panel that comes out, as content is obscured and cannot be resized to even see what is there, and no further testing or comment will help until UI scales.

**Comment by SabourDarvish** [comment]
Is there a solution? Thanks :-)

**Comment by Arthanfel** [comment]
Same issue here, may need some advice in tweaking it or even just update?
